src/utils/SuggestedCollabs.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @flow
+ * @file Suggested Collaborator utility functions
```

Comment:
It will be used by an external dependency as well unrelated to USM. This logic is currently duplicated in EUA for the create-new-folder modal as well, that was one of the primary motivations for doing this refactor. After this goes out I plan on consuming this util from there as well. Is that enough to keep it in the general utils folder?

Reply:
I see. It definitely sounds like we're in need of a new low-level component that all of these various high-level features can share.